app/models/claim.rb

```diff
@@ -136,6 +137,12 @@ class Claim < ApplicationRecord
   has_one :support_ticket, dependent: :destroy
 
   belongs_to :eligibility, polymorphic: true, inverse_of: :claim, dependent: :destroy
+
+  belongs_to :journeys_session,
```

Comment:
My thought was that we would discard the session record after submission, by copying the data over to the `Claim`. It does make sense to keep the session record however.

We need to be mindful of performance. the table will keep growing over time as it's mixed with submitted claims. If we address the table by ID exclusively it might not be a problem, but something to check.